packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/WebViewHostApiImpl.java

```diff
@@ -477,4 +501,12 @@ public void setWebChromeClient(Long instanceId, Long clientInstanceId) {
     final WebView webView = (WebView) instanceManager.getInstance(instanceId);
     webView.setWebChromeClient((WebChromeClient) instanceManager.getInstance(clientInstanceId));
   }
+
+  private static String parseNullStringIdentifier(String value) {
```

Comment:
Can you add a `@Nullable` annotation to this?

Reply:
Nice catch, thank update.